src/static/events.json

```diff
@@ -36,11 +36,11 @@
     "hostedAtUrl": "https://blockchainnigeria.group/"
   },
   {
-    "id": 4,
+    "id": 5,
```

Comment:
ah yes, because the number 4 is not approved.
I changed the logic so that all events (approved or not) count towards the ids of the events. this way the ids will not change from now on if an event goes from not approved to approved.